packages/core/data-transfer/lib/providers/local-strapi-destination-provider/index.ts

```diff
@@ -97,17 +97,16 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
   }
 
   async getConfigurationStream(): Promise<Writable> {
-    if (!this.strapi) {
-      throw new Error('Not able to stream Configurations. Strapi instance not found');
-    }
-
-    const strapi = this.strapi;
-
     return new Writable({
       objectMode: true,
-      async write(config, _encoding, callback) {
+      write: async (config: IConfiguration<any>, _encoding, callback) => {
```

Comment:
```suggestion
      write: async (config: IConfiguration, _encoding, callback) => {
```

Reply:
the problem with this, is I can't use anything from inside value object, it will just throw an error cause it's unknown, the only other thing I could do would be to type core-store and webhooks value

Reply:
At this level, I don't think it'll cause any issues though. If this is about the `restoreCoreStore` where you use `data.value`, I would suggest updating this function with `any` or a more precise type. What do you think?